locomotive.php

```diff
@@ -56,6 +59,46 @@ public function dashboard_display() {
 	}
 
 	/**
+	 * The contextual help tab display.
+	 */
+	public function dashboard_help_tab() {
+
+		// Bail if we don't have our current screen available yet.
+		if ( ! function_exists( 'get_current_screen' ) ) {
+			return;
+		}
+
+		$screen = get_current_screen();
+
+		// If any part of getting the screen isn't correct, bail.
+		if ( ! is_object( $screen ) || empty( $screen->id ) || 'tools_page_locomotive' !== $screen->id ) {
+			return;
+		}
+
+		// Load our actual help tab.
+		$screen->add_help_tab( array(
+			'id'        => 'help-overview',
+			'title'     => __( 'Overview' ),
+			'content'   => '<p>' . __( 'Locomotive is a batch processing library that can be used to write a single or set of functions to be processed across a large data set, right from the WordPress admin. You can use it to add meta values to posts based on arbitrary data, process and delete spam comments and revisions, submit posts through external API\'s, or simply change data on a large amount of posts at the same time.' ) . '</p>',
```

Comment:
We've used the "locomotive" text domain elsewhere on the site. Do we want to add them here?